dev-docs/bidders/duration.md

```diff
@@ -0,0 +1,86 @@
+---
+layout: bidder
+title: Duration Media
+description: Prebid Duration Media Bidder Adapter
+biddercode: duration
+aliasCode : nobid
+pbjs: true
+pbs: true
+media_types: banner, video
+gdpr_supported: true
+gvl_id: 816
+usp_supported: true
+schain_supported: true
+coppa_supported: true
+userId: criteo, unifiedId, id5Id
+safeframes_ok: true
```

Comment:
please indicate false as per your comments below

Reply:
@patmmccann 
I don't understand your request. Why would I change these parameters?

Reply:
@redaguermas - he's suggesting that your statement below "it is highly recommended to make sure the “Serve in Safeframe” box in creative settings is unchecked" makes it sound like you don't actually support safeframes, so `safeframes_ok` should be false.  But @patmmccann , they do also then say "If you absolutely want to run...", so perhaps they do support it somewhat grudgingly.